data.sql

```diff
@@ -34,3 +34,46 @@ VALUES
 UPDATE animals SET Species = 'digimon' WHERE Name like '%mon';
 
 UPDATE animals SET Species = 'pokemon' WHERE Species = 'unspecified';
+
+INSERT INTO owners
+  (full_name, age)
+VALUES
+  ('Sam Smith ', 34),
+  ('Jennifer Orwell', 19 ),
+  ('Bob', 45 ),
+  ('Melody Pond', 77 ),
+  ('Dean Winchester', 14 ),
+  ('Jodie Whittaker', 38 );
+
+INSERT INTO species
+  (name)
+VALUES
+  ('Pokemon'),
+  ('Digimon');
```

Comment:
![Screenshot from 2022-09-14 17-48-17](https://user-images.githubusercontent.com/93039359/190158378-7caf0eec-671c-44f8-acad-c95198b8b749.png)

- [ ] Your are inserting values into the name column which doesn't exist as you can see in the error in the above image. You have a column name full_name where you need to insert all these values :+1:
- [ ] You also need to insert the id because it is primary key so it cannot be null :+1:

Reply:
@HAMMAS-SALEEM  The ID is always generated as an identity-primary key. And on my side, it is not showing an error or null.